irods/exception.py

```diff
@@ -724,6 +724,10 @@ class NO_LOCAL_FILE_RSYNC_IN_MSI(UserInputException):
     code = -356000
 
 
+class LOCKED_DATA_OBJECT_ACCESS(SystemException):
```

Comment:
didn't see this used anywhere

Reply:
Not sure why the `code = -406000` line isn't included in the diff.
This isn't, directly, used but it allows messages to be printed out by an identifiable class name ('LOCKED_DATA_OBJECT_ACCESS') rather than by code (eg "KeyError: -406000") so that server messages can seem less obtuse.

Reply:
it's here in the diff.     all good.